crates/ruff_linter/src/rules/flake8_type_checking/rules/runtime_import_in_type_checking_block.rs

```diff
@@ -155,6 +155,14 @@ pub(crate) fn runtime_import_in_type_checking_block(
                     && binding.references().all(|reference_id| {
                         let reference = checker.semantic().reference(reference_id);
                         reference.in_typing_context() || reference.in_runtime_evaluated_annotation()
+                        // TODO: We should check `reference.in_annotated_type_alias()`
+                        //       as well to match the behavior of the flake8 plugin
+                        //       although maybe the best way forward is to add an
+                        //       additional setting to configure whether quoting
+                        //       or moving the import is preferred for type aliases
+                        //       since some people will consistently use their
+                        //       type aliases at runtimes, while others won't, so
+                        //       the best solution is unclear.
```

Comment:
Can we move this TODO to the top of the closure? (I'm also wondering if this should just be comment on the review?)